prac_06/guitars.py

```diff
@@ -0,0 +1,15 @@
+from prac_06.guitar import Guitar
+
+guitars = []
+print("My guitars!")
+name = input("Name: ")
+while name != "":
+    year = int(input("Year: "))
+    cost = float(input("Cost: $"))
+    guitars.append(Guitar(name, year, cost))
+    print("")
+    name = input("Name: ")
+print("These are my guitars: ")
+for i, guitar in enumerate(guitars):
+    vintage_string = guitar.is_vintage()
```

Comment:
This works nice and simply for you because you defined the is_vintage() method the way you did - you probably did that later once you'd tested the is_vintage calculation was working properly (I haven't checked all of your commits, sorry!) :-)